packages/hardhat-plugin/src/constants.ts

```diff
@@ -1,5 +1,5 @@
-export const UNLOCK_LATEST_VERSION = 11
-export const PUBLIC_LOCK_LATEST_VERSION = 13
+export const UNLOCK_LATEST_VERSION = 13
```

Comment:
I wonder if our "contracts" package should just have some `latest` fields?

Reply:
yes it will be the easiest and safest way. We need an alias in our exports, there is an existing opened issue on these I referenced  here